src/workflow_components/composite_action.py

```diff
@@ -95,6 +99,7 @@ class CompositeAction(GraphObject):
     using = Property()
     image = Property()
     url = Property()
+    tag = Property()
 
     steps = RelatedTo(CompositeActionStep)
 
```

Comment:
we need to verif that `_id` remains unique among same actions with different tags

Reply:
Since it is calculated from `path` property of Action, it will be since it contains the full uses string.